src/main/kotlin/example/SidebarOpenedModule.kt

```diff
@@ -1,21 +1,29 @@
 package example
 
+import mui.system.ThemeProvider
 import react.*
 
 data class SidebarOpened(
     val value: Boolean,
     val setValue: StateSetter<Boolean>,
+    val darkMode: Boolean,
```

Comment:
Single Responsibility Principle is violated here.
Theme has no relation to `SidebarOpenedContext`.